addons/mail/static/src/components/discuss/discuss.scss

```diff
@@ -1,92 +1,8 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
 .o_Discuss {
-    display: flex;
-    height: map-get($sizes, 100);
     min-height: 0;
-
-    &.o-isDeviceSmall {
-        flex-flow: column;
-        align-items: center;
-    }
 }
 
 .o_Discuss_chatWindowHeader {
     width: map-get($sizes, 100);
     flex: 0 0 auto;
 }
-
-.o_Discuss_content {
-    height: map-get($sizes, 100);
```

Comment:
I don't see `height` in classname (1st commit)

Reply:
Fixed 👍